boxsdk/object/upload_session.py

```diff
@@ -126,13 +127,18 @@ def commit(
         }
         if etag is not None:
             headers['If-Match'] = etag
-        response = self._session.post(
-            self.get_url('commit'),
-            headers=headers,
-            data=json.dumps(body),
-        )
-        if response.status_code == 202:
-            return None
+
+        try:
+            response = self._session.post(
+                self.get_url('commit'),
+                headers=headers,
+                data=json.dumps(body),
+            )
+        except BoxAPIException as box_api_exc:
+            if box_api_exc.status == 202:
```

Comment:
could use this from `http` module:
```suggestion
            if box_api_exc.status == HTTPStatus.ACCEPTED:
```

Reply:
good point